gegelatilib/include/data/demangle.h

```diff
@@ -0,0 +1,18 @@
+#ifndef GEGELATI_DEMANGLE_H
+#define GEGELATI_DEMANGLE_H
```

Comment:
use `DEMANGLE_H` for the macro to stay consistent with other files.

Reply:
My IDE was'nt properly set up I fix that here (and in the codegen)